Autopilot/Src/IMUThreadInterface.cpp

```diff
@@ -0,0 +1,12 @@
+#include "IMUThreadInterface.h"
+#include "imu.hpp"
+
+void IMUThreadInterfaceInit(void) {
+    static IMU &imuobj = BMX160::getInstance();
```

Comment:
Suggestion:

```cpp
#include "IMUThreadInterface.h"
 #include "imu.hpp"

static IMU &imuobj = BMX160::getInstance();

 void IMUThreadInterfaceInit(void) {
     imuobj.Begin_Measuring();
 }

 void IMUThreadInterfaceExecute() {
     imuobj.Begin_Measuring();
 }
```

Reply:
So I tested that and surprisingly the threads don't work when implemented that way, not sure why but I'll look into it

Reply:
sound good. Don't waste time looking into it. There are more important things than this tidbit lol